manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RakutenSymphonyKorea.SendAnywhere

```

```suggestion
PackageIdentifier: Estmob.SendAnywhere
```
Best to match publisher from control panel